README.md

```diff
@@ -204,12 +204,13 @@ If you don't feel like getting your hands dirty with code there are still other
 
 ## Output formats
 
-`reek` supports 3 output formats:
+`reek` supports 5 output formats:
 
 * plain text (default)
 * html (`--format html`)
 * yaml (`--format yaml`)
 * json (`--format json`)
+* xml  (`--format xml`)
```

Comment:
It's first XML based format for Reek since the beginning of Reek. I believe there is low chance to get another XML based format introduced in near feature for Reek. In other hand everyone knows what is XML and only relatively small amount of peeps knows what is Checkstyle, so I suggest to stay with XML name.